sms-checker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM public.ecr.aws/lambda/python:latest
+
+WORKDIR ${LAMBDA_TASK_ROOT}
+
+COPY requirements.txt .
+RUN pip install -r requirements.txt 
+
+COPY notification_lambda.py .
+COPY checker.py .
+COPY message.py .
+
+CMD [ "notification_lambda.lambda_handler" ]
```

Comment:
Functional docker container for ECR that  imports all necessary files and installs dependencies while running the actual pipeline in the end. LGTM.